sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/db/derived/TxOutputsView.kt

```diff
@@ -71,7 +71,9 @@ internal class TxOutputsView(private val sqliteDatabase: SupportSQLiteDatabase)
                 val toAddressIndex = it.getColumnIndex(TxOutputsViewDefinition.COLUMN_STRING_TO_ADDRESS)
 
                 if (!it.isNull(toAccountIndex)) {
-                    TransactionRecipient.Account(Account(it.getInt(toAccountIndex)))
+                    // TODO [#1644]: Refactor Account ZIP32 index across SDK
+                    // TODO [#1644]: https://github.com/Electric-Coin-Company/zcash-android-wallet-sdk/issues/1644
+                    TransactionRecipient.Account(0)
```

Comment:
Fixing this requires #1640 to be updated once the corresponding changes to the Rust backend are made.